keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -6,6 +6,7 @@ import java.time.Duration.ZERO
 sealed class ClusterDeployStrategy {
   open val isStaggered: Boolean = false
   open val stagger: List<StaggeredRegion> = emptyList()
+  abstract val considerOnlyAmazonHealth: Boolean
```

Comment:
I considered moving this off the deploy strategy, but it feels like it makes sense here. It's relevant only to launching tasks (so Orca knows what kind of health to wait for) and determining if a new server group is deployed successfully (i.e. all the instances are healthy).

Reply:
My suggestion is `considerOnlyInstanceHealth`.